.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Contract Development
 
-on: 
+on:
```

Comment:
@maurolacy can you install an https://editorconfig.org/ plugin in your editor to avoid such diffs?

Reply:
Sure.